### PR TITLE
 Add email , password and confirm password fields to create user form

### PR DESCRIPTION
**Pull request by lakhina**
-  Update the "Create User" form to include email, password and verify password fields 
-  Show an error if the password and verify password fields don't match

### CODE REVIEW

**Comment by lakhina** [comment]
@TimMoore  I just opened a new PR . Please review.

File: web-gateway/app/controllers/Main.java [hunk 1]
```diff
@@ -51,10 +52,12 @@ public Main(Configuration config, MessagesApi messagesApi, UserService userServi
                 loadNav(userId).thenCompose(nav -> {
                     Form<CreateUserForm> form = formFactory.form(CreateUserForm.class).bindFromRequest(ctx.request());
                     if (form.hasErrors()) {
-                        return CompletableFuture.completedFuture(ok(views.html.createUser.render(showInlineInstruction, form, nav)));
+                        return CompletableFuture.completedFuture(ok(views.html.createUser.render(showInlineInstruction, form,Optional.empty(), nav)));
                     }
-
-                    return userService.createUser().invoke(new User(form.get().getName())).thenApply(user -> {
+                    if(!(form.get().getPassword().equals(form.get().getConfirmPassword()))){
```
Comment (lakhina) [inline web-gateway/app/controllers/Main.java]: @TimMoore  Ok, then I'll start implementing the suggested changes

**Comment by lakhina** [comment]
I am up with leaning this PR. Will complete that in few hours

**Comment by lakhina** [comment]
@TimMoore @ignasi35 I have completed `change 1` in this PR and have removed all the code which were touching anything extra. I will complete change 2 and 3 in next PRs. 

I will be a bit stuck until this PR is merged so please review when you get time.

**Comment by TimMoore** [comment]
Thanks, @lakhina ... I'll review it now.

**Pull request merged**